routing/router.go

```diff
@@ -1326,25 +1342,21 @@ func (r *ChannelRouter) processUpdate(msg interface{},
 				fallthrough
 
 			case strings.Contains(err.Error(), "out of range"):
-				// If the funding transaction isn't found at all, then
-				// we'll mark the edge itself as a zombie so we don't
-				// continue to request it. We use the "zero key" for
-				// both node pubkeys so this edge can't be resurrected.
-				var zeroKey [33]byte
-				zErr := r.cfg.Graph.MarkEdgeZombie(
-					msg.ChannelID, zeroKey, zeroKey,
-				)
+				// If the funding transaction isn't found at
```

Comment:
Should also add here the case where the transaction index is greater than the number of transactions in the block.

Reply:
This is caught as is, since the "out of range" sub-string will match this case.